apps/src/lab2/views/components/AiTutor2Response.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {useCallback, useEffect, useRef} from 'react';
+
+import ChatMessage from '@cdo/apps/aiComponentLibrary/chatMessage/ChatMessage';
+import {Role} from '@cdo/apps/aiComponentLibrary/chatMessage/types';
+import AiTutor2Manager from '@cdo/apps/lab2/ai/AiTutor2Manager';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+import moduleStyles from './AiTutor2Response.module.scss';
+
+interface AiTutor2ResponseProps {
+  type: 'user' | 'hint';
```

Comment:
eventually these'll probably be constants? something like `AiTutorModes.hint`?

Reply:
Good question.  Added a type, for starters.